content/terraform/v1.12.x/docs/language/modules/develop/providers.mdx

```diff
@@ -72,7 +72,7 @@ however, specify any of the configuration settings that determine what remote
 endpoints the provider will access, such as an AWS region; configuration
 settings come from provider _configurations_, and a particular overall Terraform
 configuration can potentially have
-[several different configurations for the same provider](/terraform/language/providers/configuration#alias-multiple-provider-configurations).
+[several different configurations for the same provider](/terraform/language/block/provider#using-an-alternate-provider-configuration).
```

Comment:
```suggestion
[several different configurations for the same provider](/terraform/language/block/provider#select-an-alternate-provider-configuration).
```
You don't have to use the heading I suggested, just something that follows the style guidance.